src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1812,7 +1822,7 @@ void TorrentHandle::flushCache()
 
 QString TorrentHandle::toMagnetUri() const
 {
-    return m_torrentInfo.toMagnetUri();
```

Comment:
Suddenly useful? I think the smart linker will not link it.
But I can remove. If you say so.